apps/src/javalab/JavalabView.jsx

```diff
@@ -66,15 +67,32 @@ class JavalabView extends React.Component {
   static propTypes = {
     onMount: PropTypes.func.isRequired,
     onContinue: PropTypes.func.isRequired,
+    suppliedFilesVersionId: PropTypes.string,
 
     // populated by redux
     isProjectLevel: PropTypes.bool.isRequired,
     isReadOnlyWorkspace: PropTypes.bool.isRequired,
     appendOutputLog: PropTypes.func
   };
 
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      loading: true,
+      loadSuccess: null
+    };
+  }
```

Comment:
you don't need the constructor here. you can initialize state in the root of the component like this:

```javascript
state = {
  loading: true,
  loadSuccess: null
};
```